src/storage/write_ahead_log/log_manager.cpp

```diff
@@ -64,4 +132,59 @@ void LogManager::SerializeRecord(const terrier::storage::LogRecord &record) {
   }
 }
 
+void LogManager::WriteToDiskLoop() {
```

Comment:
I suspect you want to decouple the disk writer from the serializer into a separate object for better understandability